### DISCUSSION TITLE
Who should be credited in CC license?

### DISCUSSION THREAD

**Discussion by matkoniecz** [osmandapp/OsmAnd#15038] [Q&A]
For https://github.com/osmandapp/Osmand/blob/master/LICENSE#L39 - who should be attributed in this CC license?

> OsmAnd BV (Amstelveen, Netherlands - KvK 62066714, BTW 854627704B01)

> OsmAnd BV

committer to a specific file?

Concerns https://wiki.openstreetmap.org/wiki/File:Osmand_information_route_marker.svg and https://wiki.openstreetmap.org/wiki/File:Osmand_information_guidepost.svg

**Comment by vshcherb** [comment]
As I understand it was drawn by @xmd5a2 https://github.com/osmandapp/OsmAnd-resources/blob/master/icons/svg/tourist_colored/information_route_marker.svg as part of his work for OsmAnd BV. We didn't define separate license https://github.com/osmandapp/OsmAnd-resources/tree/master/icons/svg except icons8 we have no ownership.
